en.md

```diff
@@ -75,13 +75,13 @@ This data is anonymised after a short space of time, generally 7 days. The anony
 
 ### b. Third party integrations
 
-We use a 3rd party service called "HockeyApp" to collect and analyse crash reports of our app. 
+We use a 3rd party service called "App Center" to collect and analyze crash reports of our app.
```

Comment:
Shall we use proper quotes like on L185, for consistency?

Reply:
I think I don't mind either way